broadway_api/flags.py

```diff
@@ -0,0 +1,124 @@
+from flagset import Flag
+
+app_flags = {
```

Comment:
Question: Where did we declare the required argument of the config path? Can not find it here and I think this is what FlagSet uses to parse command-line arguments? Or is that first required argument baked into FlagSet?

Reply:
oh it's generated by `flagset` automatically. there is no way to declare positional arguments rn

Reply:
gotcha